user/views.py

```diff
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+import json, bcrypt, jwt
 
-# Create your views here.
+from django.shortcuts       import render
+from django.views           import View
+from django.http            import JsonResponse, HttpResponse
+
+from user.models            import User
+from my_settings            import SECRET_KEY, AL
```

Comment:
```suggestion
from my_settings            import SECRET_KEY, ALGORITHM
```
`AL`은 다소 명확하지 않은 변수 이름입니다.
꼭 필요한 경우가 아니라면 줄임말은 되도록 쓰지 않는 것을 추천드립니다 :)